cmd/run.go

```diff
@@ -20,7 +20,9 @@ import (
 
 func dockerStop(imageName string, dryrun bool) error {
 	cmdName := "docker"
-	cmdArgs := []string{"stop", imageName}
+	signalInterval := "10" // numnrt of seconds to wait prior to sending SIGKILL
+	Info.logf("Performing docker stop. Shutdown will occur in %s seconds", signalInterval)
```

Comment:
Let's remove this message. The execAndWait will already log the command `docker stop -t 10` so the user can figure out what that means.

Reply:
changed.